platform/lib/utils/imageOptimizer.js

```diff
@@ -25,8 +37,22 @@ const config = require('@lib/config');
  * @param {Integer} width  - the target width
  */
 function imageOptimizer(src, width) {
+  // Do not optimize non amp.dev URLs, as Thumbor will not process them
+  // also do not try to optimize SVGs
+  if (
+    src.startsWith('http://') ||
+    src.startsWith('https://') ||
+    src.endsWith('.svg')
+  ) {
+    return null;
+  }
+
   const imageUrl = new URL(src, config.hosts.platform.base);
   imageUrl.searchParams.set('width', width);
+
+  const hash = imageIndex[join('dist/', imageUrl.pathname)];
```

Comment:
Not super happy about encoding the build dir here (have we got a variable for this?)